debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java

```diff
@@ -673,6 +673,18 @@ else if (inconsistentSchemaHandlingMode == EventProcessingFailureHandlingMode.WA
             }
         }
         else {
+            if (tableId == null) {
```

Comment:
I'll move this into an if block checking the log level, so to avoid the look-up if we're not on DEBUG level.